src/static/client/app/signup/signup.component.scss

```diff
@@ -1,110 +1,130 @@
+  align-items: center;
+  height: 100vh;
+
+  .button-container {
     display: flex;
+    flex-direction: column;
     justify-content: center;
     align-items: center;
-    height: 100vh;
+  }
 
-    .button-container {
-      display: flex;
-      justify-content: space-between;
-    }
-    
-    #icon {
-      width: 150px;  // Adjust as needed to change the size of the logo.
-      height: auto;
-      display: block;
-      margin: 0 auto 20px auto; // This centers the image and adds space below it.
+  #icon {
+    width: 60px;
+    height: auto;
+    display: block;
+    margin: 0 auto 20px;
+  }
+
+  .wrapper {
+    max-width: 400px;
+    background-color: #fff;
+    border-radius: 10px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
+    padding: 30px;
+    width: 100%;
+    max-width: 350px;
+
+    .h2 {
+      text-align: center;
+      margin-bottom: 24px;
     }
-  
-    .wrapper {
-      max-width: 400px;
-      background-color: #fff;
-      border-radius: 10px;
-      box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
-      padding: 30px;
-      width: 100%;
-      max-width: 350px;
-  
-      .h2 {
-        text-align: center;
-        margin-bottom: 24px;
-      }
-  
-      #formContent {
-        display: flex;
-        flex-direction: column;
-  
-        .form-label {
-          display: block;
-          font-size: 14px;
-          font-weight: 600;
-          color: #4a4a4a;
+
+    #formContent {
+      display: flex;
+      flex-direction: column;
+
+      .fadeIn.first {
+        margin-top: 30px;
+
+        .info {
+          font-size: 14px; 
           margin-bottom: 10px;
         }
-        
-  
-        .form-input {
-          width: 100%;
-          padding: 10px;
-          font-size: 14px;
-          border: 1px solid #e0e0e0;
-          border-radius: 5px;
-          outline: none;
-          transition: border-color 0.3s;
-          margin-bottom: 20px;
-        
-          &:focus {
-            border-color: #007bff;
-          }
-        }
-        .check-lg {
-          margin-bottom: 20px;
-        }
-  
-        .form-button {
-          padding: 10px;
-          font-size: 16px;
-          font-weight: 600;
-          border: none;
-          border-radius: 5px;
-          cursor: pointer;
-          outline: none;
-          transition: background-color 0.3s;
-          margin-bottom: 10px;
-        
-          &:disabled {
-            background-color: #ccc;
-          }
+      }
+
+      .form-label {
+        display: block;
+        text-align: center;
+        font-size: 18px;
+        font-weight: 600;
+        color: #4a4a4a;
+        margin-bottom: 10px;
+      }
+
+      .form-input {
+        width: 100%;
+        padding: 10px;
+        font-size: 14px;
+        border: 1px solid #e0e0e0;
+        border-radius: 5px;
+        outline: none;
+        transition: border-color 0.3s;
+        margin-bottom: mb-3;  
```

Comment:
So mb-3 is a bootstrap class. So it should be added in the html directly like: 
class="mb-3 form-input". Adding it like this will not work to my knowledge. If there is any documentation on using mb-3 like this in a css file, shoot it my way.